action.yml

```diff
@@ -32,6 +32,9 @@ inputs:
 runs:
   using: "composite"
   steps:
+      - name: Deprecation Notice
```

Comment:
FYI ... Looks like the indention is incorrect here. Leading to the following errors. Workaround is easy though...we're just migrating to v2.

```
linear-b/gitstream-github-action/v1/action.yml: (Line: 36, Col: 7, Idx: 1019) - (Line: 36, Col: 7, Idx: 1019): While parsing a block collection, did not find expected '-' indicator.
```

```
linear-b/gitstream-github-action/v1/action.yml (Line: 35, Col: 9): There's not enough info to determine what you meant. Add one of these properties: run, shell, uses, with, working-directory
```